apps/system/js/statusbar.js

```diff
@@ -160,91 +74,105 @@ var StatusBar = {
+    <div id="statusbar-call-forwardings" class="sb-icon-call-forwarding"
+      hidden role="presentation"></div>
+    <div id="statusbar-geolocation" class="sb-icon sb-icon-geolocation"
+      hidden role="listitem" data-l10n-id="statusbarGeolocation"></div>
+    <div id="statusbar-recording" class="sb-icon sb-icon-recording"
+      hidden role="listitem" data-l10n-id="statusbarRecording"></div>
+    <div id="statusbar-mute" class="sb-icon sb-icon-mute"
+      hidden role="listitem"></div>
+    <div id="statusbar-usb" class="sb-icon sb-icon-usb" hidden role="listitem"
+      data-l10n-id="statusbarUsb"></div>
+    <!-- See note on <img> above. -->
+    <img id="statusbar-download"
+      src="style/statusbar/images/system-downloads.png"
+      class="sb-icon-download" hidden role="listitem"
+      data-l10n-id="statusbarDownload">
+    <div id="statusbar-emergency-callback"
+      class="sb-icon sb-icon-emergency-callback"
+      hidden role="listitem"
+      data-l10n-id="statusbarEmergencyCallback"></div>
+    <div id="statusbar-debugging" data-icon="bug"
+      class="sb-icon sb-icon-debugging" hidden role="listitem"></div>
+    <!-- Connectivity -->
+    <div id="statusbar-nfc" class="sb-icon sb-icon-nfc" hidden role="listitem"
+      data-l10n-id="statusbarNfc"></div>
+    <div id="statusbar-bluetooth-transfer"
+      class="sb-icon sb-icon-bluetooth-transfer"
+      role="listitem" hidden
+      data-l10n-id="statusbarBluetoothTransfer"></div>
+    <div id="statusbar-bluetooth"
+      class="sb-icon sb-icon-bluetooth" hidden role="listitem"></div>
+    <div id="statusbar-tethering"
+      class="sb-icon sb-icon-tethering" hidden role="listitem"></div>
+    <!-- HACK: We use images instead of divs to enforce allocation of a
+         dedicated layer just for this animated icons, remove after
+         https://bugzil.la/717872 gets fixed -->
+    <img id="statusbar-network-activity"
+      src="style/statusbar/images/network-activity.png"
+      class="sb-icon-network-activity" hidden role="listitem"
+      data-l10n-id="statusbarNetworkActivity">
+    <div id="statusbar-mobile-connection"
+      class="sb-icon-mobile-connection" hidden role="presentation"></div>
+    <div id="statusbar-wifi" class="sb-icon sb-icon-wifi"
+      data-level="4" hidden role="listitem"></div>
+    <div id="statusbar-airplane-mode"
+      class="sb-icon sb-icon-airplane-mode" hidden role="listitem"
+      data-l10n-id="statusbarAirplaneMode"></div>
+    <!-- General -->
+    <div id="statusbar-battery" class="sb-icon sb-icon-battery"
+      role="listitem"></div>
+    <div id="statusbar-time" class="sb-icon-time" role="listitem"></div>`;
+  },
+  /* jshint ignore: end */
+
+  renderIcons: function() {
+    this.statusbarIconsMax.insertAdjacentHTML('afterbegin', this.iconView());
+    Service.register('getIcon', this);
+    Service.register('startIcon', this);
+    Service.register('stopIcon', this);
```

Comment:
Looks like nobody is using `startIcon`/`stopIcon` (and that they're not implemented), but still reading :)